bin/pca-import

```diff
@@ -105,7 +105,7 @@ def output_assessment_template():
     assessment["team_lead"] = "Test Testerton"
     assessment["start_time"] = "2017-01-01"
     assessment["end_time"] = "2017-01-21"
-    print(util.to_json(assessment))
+    print(json.dumps(assessment, sort_keys=False, indent=4))
```

Comment:
I think that instead of how you have approached this the `util.to_json()` function should have a `sort_keys` keyword argument added that defaults to `True` to replicate existing behavior. Then you can override it in cases such as this by calling `util.to_jason(assessment, sort_keys=False)`. You lose the [custom JSON handler](https://github.com/cisagov/pca-core/blob/2de0f306026c79ac7b7a0d765a2191c37c6fdcc4/pca/util/util.py#L79-L97) the way you have resolved this issue.

So `util.to_json()` would become:

```python
def to_json(obj, sort_keys=True):
    return json.dumps(obj, sort_keys=sort_keys, indent=4, default=custom_json_handler)
```

and you could update this call appropriately.

Reply:
The custom json handler does nothing for me. It is unnecessary for this statement for what Im trying to do.

Reply:
See here for changes 4050100